dotnet-web/README.md

```diff
@@ -0,0 +1,36 @@
+---
+title: .NET
+description: Set up a CI pipeline for a .NET web app with restore, test and build commands
+tags: ["CI", ".NET"]
+author: Buildkite
+---
+
+# CI for .NET with Buildkite
+
+This template gives you a continuous integration (CI) pipeline that automates installing dependencies, run unit test and build.
+
+At a glance:
+
+- For .NET projects
+- Requires [Docker](https://docs.docker.com/get-docker/)
+- Automates build, test, and deployment
```

Comment:
I reckon we need to be more specific on what this pipeline is using for building, testing, (and deploying). Is it deploying? It's not, we should remove that part.

Reply:
Specified this pipeline uses the .NET SDK CLI to run restore, build and test.